src/server/json_family.cc

```diff
@@ -106,6 +107,56 @@ ParseResult<WrappedJsonPath> ParseJsonPath(std::string_view path) {
 
 }  // namespace json_parser
 
+namespace reply_generic {
+
+template <typename T> void Send(RedisReplyBuilder& rb, T value) = delete;
```

Comment:
Do you call `Send` in some other, templated code?
If not I do not see much value to declare it templated here.